tiny_skia/src/text.rs

```diff
@@ -405,7 +405,10 @@ impl Cache {
         }
 
         if let hash_map::Entry::Vacant(entry) = self.entries.entry(hash) {
-            let metrics = cosmic_text::Metrics::new(key.size, key.size * 1.2);
+            let metrics = cosmic_text::Metrics::new(
+                key.line_height,
+                (key.line_height * 1.2).max(f32::MIN_POSITIVE),
```

Comment:
Almost! This should look the same as the `wgpu` code:

```suggestion
                key.size,
                key.line_height.max(f32::MIN_POSITIVE),
```

You can test the `tiny-skia` backend by setting the `ICED_BACKEND=tiny-skia` env variable.

Reply:
Alright, I just committed the requested changes.

Also, I've been recompiling `iced` with `default-features=false` to test Tiny Skia. :sweat_smile: That env var is very helpful.